hw1/02-palindrome.js

```diff
@@ -1,3 +1,36 @@
-const elem = document.querySelector('input');
+const section = document.querySelector("section");
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+//Input event listener
+elem.addEventListener("input", handleInput);
+
+//Create a new div for the output
+const outputDiv = document.createElement("div");
+outputDiv.setAttribute("id", "output");
+section.append(outputDiv);
+
+//Handle the event listener
+function handleInput(event) {
+  event.preventDefault();
+  const input = event.target.value;
+  let reverseInput = input.split("").reverse().join("");
+  console.log(input);
+
```

Comment:
The assignment sheet asks to error check for non-number input. Probably want to check for that somewhere.